dist/npm-to-yarn.mjs

```diff
@@ -485,6 +485,133 @@ function npmToPnpm(_m, command) {
     }
 }
 
+function convertInstallArgs(args) {
+    // bun uses -g and --global flags
+    // bun mostly conforms to Yarn's CLI
+    return args.map(function (item) {
+        switch (item) {
+            case '--save-dev':
+            case '--development':
+            case '-D':
+                return '--dev';
+            case '--save-prod':
+            case '-P':
+                return '--production';
+            case '--no-package-lock':
+                return '--no-save';
+            case '--save-optional':
+            case '-O':
+                return '--optional';
+            case '--save-exact':
+            case '-E':
+                return '--exact';
+            case '--save':
+            case '-S':
+                // this is default in bun
+                return '';
+            case '--global':
+            case '-g':
+                return '--global';
+            default:
+                return item;
+        }
+    });
+}
+function npmToBun(_m, command) {
+    var args = parse((command || '').trim());
+    var index = args.findIndex(function (a) { return a === '--'; });
+    if (index >= 0) {
+        args.splice(index, 1);
+    }
+    var cmd = 'bun';
+    switch (args[0]) {
+        case 'install':
+        case 'i':
+            if (args.length === 1) {
+                args = ['install'];
+            }
+            else {
+                args[0] = 'add';
+            }
+            args = convertInstallArgs(args);
+            break;
+        case 'uninstall':
+        case 'un':
+        case 'remove':
+        case 'r':
+        case 'rm':
+            args[0] = 'remove';
+            args = convertInstallArgs(args);
+            break;
+        case 'cache':
+            if (args[1] === 'clean') {
+                args = ['pm', 'cache', 'rm'];
+                args = args.concat(args.slice(2));
```

Comment:
this is not correct as rm is going to be present 2 times

i'm not sure about syntax, but maybe soemthign like this?
```suggestion
                args = ['pm', 'cache', 'rm'].concat(args.slice(2));
```

Reply:
Good catch